k8s/deploy-storageos/etcd-tls/cleanup-etcd.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ETCD_NS=etcd
+ROLE_NAME=etcd-operator
+ROLE_BINDING_NAME=etcd-operator
+ETCD_OPERATOR_ROOT="./etcd-deployment/etcd-operator"
+
+# Do not delete etcd if StorageOS is running as this will destroy the StorageOS
+# cluster
+if kubectl -n storageos get pods 2>/dev/null | grep -q "STATUS"; then
+    printf "\e[31m Pods are running in the StorageOS namespace. Removing etcd resources will break the cluster so run ./storageos-deployment/cleanup.sh first\e[0m"
```

Comment:
The storageos-deployment/cleanup.sh doesn't exist.

Reply:
Fixed path and made the error message more helpful. Now it outlines what to do for operator and manifest installations.